tests/selenium/spec/quickstarts-spec.js

```diff
@@ -30,7 +30,7 @@ describe('quickstarts page default selections spec', () => {
   const quickstartsPage = new QuickStartsPage('/quickstart/');
 
   beforeEach(util.itHelper(async () => {
-    await quickstartsPage.navigate('/quickstart/');
+    await quickstartsPage.navigate('/documentation/');
```

Comment:
Why is this change here?  are we just avoiding being on `/quickstart/` to start each test?

Reply:
Yes. It seems that the navigation redirects don't always work right if it thinks it's already on the right page.